ClinGen/clincoded
# Connects to #759. Connects to #761. Connects to #770.

This PR consists of changes pertinent to #759, #761 and #770.

**Known issues:**
1. Currently, after the modal is closed upon associating the interpretation with disease, the disease id/term is not immediately reflected in the interpretation record slugline under "My interpretations" in the static header. Refreshing the page would invoke the disease id/term to be displayed.
2. After merging with the latest `dev`, I noticed that the criteria bar now wraps to a second line when the interpretation is active.

**Steps to test:**
1. Login and enter ClinVar ID 55980 via the variant selection interface.
2. In the evidence only view, expect to see the "Start new interpretation" button when there is not associated interpretation.
3. Proceed to the "Population" or "Predictors" tab, click on the "Start new interpretation" button. Expect to see 1) staying on the "Population" or "Predictors" tab instead of navigating back to "Basic Info" tab; 2) the display of the "Associate with disease" button.
4. Navigate back to the variant selection interface and enter the same ClinVar ID 55980. Upon returning to the evidence only view, expect to see the "Continue interpretation" button.
5. Click on the "Continue interpretation" button and expect to see the active interpretation and "Associate with disease" button appearing above the criteria bar.
6. Click on the "Associate with disease" button and expect to see the modal.
7. Enter "@@@" and click the OK button. Expect to see error in the modal.
8. Enter "orpha777" and click the OK button. Expect to see the modal to be closed and the subtitle in the header being updated with the disease term. And expect to see the "Associate with disease" button to be changed "Edit Disease" as a result.
